packages/documentation/copy/en/handbook-v2/Everyday Types.md

```diff
@@ -156,7 +156,7 @@ names.forEach((s) => {
 
 Even though the parameter `s` didn't have a type annotation, TypeScript used the types of the `forEach` function, along with the inferred type of the array, to determine the type `s` will have.
 
-This process is called _contextual typing_ because the _context_ that the function occurred in informed what type it should have.
+This process is called _contextual typing_ because the _context_ that the function occurred informs what type it should have.

```

Comment:
```suggestion
This process is called _contextual typing_ because the _context_ that the function occurred within informs what type it should have.

```